### ISSUE TITLE
no lights visible although "present"

### ISSUE THREAD

**Issue by linuzer** [diyhue/diyHue#375]
HI,
I have a very strange situation, I'm using the latest docker image and after start-up I tried to import the lights from the original HueBridge. Although it stated having read the lights, there are no lights visible.
But checking the config-file, all lights are there. And when I switch the "Turn all on" button, all lights are really going on. So this tells me, the lights are really there, just the webpage doesn't show me anything...  very strange!

Do you have any idea?

**Comment by Gizmo-Ger** [comment]
I have a similar problem. Although I´m using both docker for diyhue and deconz with IKEA tradfri lights. So far I can link them and use Hue Essentials to control them, but I don´t see any light in the WebGUI of diyhue and I can´t switch all lights on. The lights show all up in config.json

**Comment by mariusmotea** [comment]
In the webpage are displayed only the groups with the lights from them. If you did not create a group there is nothing to display. Just use an Hue app to create the groups.

**Comment by linuzer** [comment]
OK, I was able to solve this:

Since I was also unable to connect the original Hue-App to the diyHue bridge, I deleted the app completely and reinstalled it. Then -- during connection to the diyHue, the app asked to create a new room (which I did) and assigned all lights from the diyHue to that room. Only **_then_** the lights appeared also on the diyHue website.

So thanks, @mariusmotea, your comment kind-of described the actual problem. I'm not sure whether this is actually a bug or a feature of diyHue, but in my opinion, diyHue should be able to create a default room for the lights on-the-fly by itself -- independently whether of not the Hue-App is already connected.
